.github/workflows/backtrace-commit-fix.yml

```
@@ -0,0 +1,26 @@
+name: 'Backtrace issue commit correction'
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  backtrace_commit_correction:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      ISSUE: ${{ github.event.issue.html_url }}
+    if: |
+      contains(github.event.issue.labels.*.name, 'backtrace.io')
+    steps:
+      - name: Modify issue contents
+        env:
+          BODY: ${{ github.event.issue.body }}
+        id: fix
+        run: |
+          changed_body=$(echo "$BODY" | sed '/<strong>commit<\/strong>/,/<\/li>/ s/\(<code>\)\(.*\)\(<\/code>\)/\2/g')
```

This line can be made a bit shorter by removing the unnecessary groups in the regex, and passing BODY into sed without piping from echo.
```suggestion
          changed_body=$(sed '/<strong>commit<\/strong>/,/<\/li>/ s/<code>\(.*\)<\/code>/\1/' <<< "$BODY")
```

---

While its shorter its also harder to read in my opinion, makes it quite clear that the output is piped and then processed by sed, the suggestion is harder to understand until you reach the end of the line.

---

```suggestion
          changed_body=$(echo "$BODY" | sed '/<strong>commit<\/strong>/,/<\/li>/ s/<code>\([0-9a-f]{7,8}\)<\/code>/\1/')
```